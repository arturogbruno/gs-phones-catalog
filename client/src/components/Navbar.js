import React from "react";

const Navbar = ({ category }) => {
    return (
        <div className="Navbar">
            <img src="https://images.ctfassets.net/5gv1edeicqfs/2VJg7fkuNaAUqUawUuckk4/66298626f95b05550594405b2e5d39b9/gs-logo-inv.png" alt="GS logo" />
            <h3>{category.charAt(0).toUpperCase().concat(category.slice(1))} Catalog</h3>
        </div>
    )
}

export default Navbar;