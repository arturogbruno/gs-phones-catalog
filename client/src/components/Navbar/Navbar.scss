.Navbar {
    background-color: #fff;
    width: 100%;
    height: 80px;
    display: flex;
    justify-content: center;
    margin-bottom: 50px;
    box-shadow: 0px 5px 10px 0px rgba(0,0,0,0.5);
    
    a {
        color: rgb(35, 35, 35);
        display: flex;
        align-items: center;
        text-decoration: none;
        transition: all .2s ease-in-out;

        &:hover {
            transform: scale(1.05);
        }
    }

    .logo {
        height: 3rem;
        margin-right: 1rem;
    }
}
