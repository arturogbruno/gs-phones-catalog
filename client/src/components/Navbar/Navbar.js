import React from "react";
import { Link } from "react-router-dom";
import "./Navbar.scss";

const Navbar = ({ category }) => {
    return (
        <div className="Navbar">
            <Link to="/">
                <img className="logo" src="https://images.ctfassets.net/5gv1edeicqfs/48EM0LU3Z6gWkQCcCaeoq2/704ea273b5d50d09ff450a5ceaa74631/guidesmiths-logo.png" alt="GS logo" />
                <h2>{category.charAt(0).toUpperCase().concat(category.slice(1))} Catalog</h2>
            </Link>
        </div>
    )
}

export default Navbar;