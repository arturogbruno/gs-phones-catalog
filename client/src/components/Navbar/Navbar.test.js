import React from "react";
import { shallow } from "enzyme";
import Navbar from "./Navbar";

describe("testing Navbar component", () => {
    it("should render Navbar component", () => {
        expect(shallow(<Navbar category="test" />)).toMatchSnapshot();
    });
})
