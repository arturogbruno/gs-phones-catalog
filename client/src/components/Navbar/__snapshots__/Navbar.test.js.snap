// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Navbar component should render Navbar component 1`] = `
<div
  className="Navbar"
>
  <Link
    to="/"
  >
    <img
      alt="GS logo"
      className="logo"
      src="https://images.ctfassets.net/5gv1edeicqfs/48EM0LU3Z6gWkQCcCaeoq2/704ea273b5d50d09ff450a5ceaa74631/guidesmiths-logo.png"
    />
    <h2>
      Test
       Catalog
    </h2>
  </Link>
</div>
`;
